use-contract.ts

```diff
@@ -10,9 +10,19 @@ export function useContract(contractAddress: string): ContractHook {
   const [error, setError] = useState<string>();
   const [storage, setStorage] = useState();
   const [loading, setLoading] = useState(false);
+  const [refreshStorageInterval, setRefreshStorageInterval] = useState(
+    undefined
+  );
 
   useEffect(() => {
     loadStorage(contract);
+    clearInterval(refreshStorageInterval);
+    // sets interval
+    const interval = setInterval(async () => loadStorage(contract), 6000);
```

Comment:
isn't 6 seconds too low? I believe a minute is enough, as the blockchain updates every one minute, no?